### ISSUE TITLE
Retina Ready option not saving

### ISSUE THREAD

**Issue by Abelc01** [CodeCabin/wp-google-maps#979]
When you create/edit a marker and you enable the retina ready option, the option will not be enabled once you have refreshed the page or clicked on "Save Map".

**Comment by dylank6169** [comment]
Had to debug this for a customer as he urgently needed it, and I found the cause, so am just sharing the fix I added for the client:
in `class.pro-marker.php` around line 328, there is the following code:
`case "retina":
				$this->_icon->retina = ($value == 1);
				if($this->_icon->isDefault)
					$value = "";
				else
					$value = json_encode($this->_icon);
				$this->updateIcon();

				break;`

The issue is at the line `$value = json_encode($this->_icon);`. It is setting the value of the `retina` column to the JSON icon value if it is not the default, but the `retina` field is a tinyint type and is not saving as a result. Setting it to an int value if retina is true or false fixes the issue here.
